plasma_framework/package.json

```diff
@@ -27,14 +27,15 @@
     "ethereumjs-util": "^6.1.0",
     "openzeppelin-test-helpers": "^0.4.3",
     "rlp": "^2.2.3",
+    "run-with-testrpc": "^0.3.1",
     "seedrandom": "^3.0.5",
     "solhint": "^2.2.0",
-    "solidity-coverage": "^0.7.2",
-    "truffle": "5.1.36"
+    "solidity-coverage": "^0.7.12",
+    "truffle": "^5.1.48"
   },
   "scripts": {
     "test": "truffle test",
-    "coverage": "truffle run coverage",
+    "coverage": "node_modules/.bin/truffle --max-old-space-size=4096 run coverage",
```

Comment:
Were there out of memory errors when running coverage?

Reply:
yes, `heap out of memory` after I bumped coverage to latest.